### PR TITLE
11302022 as incident response approval email

### PR DESCRIPTION
**Pull request by AcceleryntSecurityDev**
Change(s):
New Playbook

Reason for Change(s):
New Playbook

Testing Completed:
Yes

### CODE REVIEW

**Comment by lior-tamir** [comment]
Hi,

1. The name of the playbook may be too general - incident response approval email may be relevant to multiple scenarios.
2. Why is Key Vault required? Can managed identity replace it?

**Comment by AcceleryntSecurityDev** [comment]
@lior-tamir 
1. I will update the playbook name. 
2. The key vault is used to store the client secret of the app registration used for http callouts to the graph api- there is not a connector for this, so managed identity cannot be used.

**Comment by v-spadarthi** [comment]
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

**Comment by v-mchatla** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

**Comment by v-spadarthi** [comment]
Hi @lior-tamir,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks
@manishkumar1991 / @rahul0216 : Please have a look and provide your feedback. Thanks!

**Comment by v-atulyadav** [comment]
Hi @lior-tamir, waiting for your response for author's comment. Thanks

**Comment by v-mchatla** [comment]
Hi @lior-tamir, @manishkumar1991, @rahul0216,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-mchatla** [comment]
Hi @lior-tamir, @manishkumar1991, @rahul0216,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-atulyadav** [comment]
Hi @lior-tamir, @manishkumar1991, @rahul0216,
Can you please review the comments from author and suggest if there is any alternate to this approach.
Thanks

**Comment by v-mchatla** [comment]
Hi @manishkumar1991, @rahul0216,
Can you please go through the authors comments and suggest if there is feasibility to use managed identity. 
Thanks

**Comment by manishkumar1991** [comment]
> Hi @manishkumar1991, @rahul0216, Can you please go through the authors comments and suggest if there is feasibility to use managed identity. Thanks

I will review it , please allow some time

File: Playbooks/AS-Incident-Response-Approval-Email/README.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+The resources should take around a minute to deploy. Once the deployment is complete, you can expand the "**Deployment details**" section to view them.
+Click the one corresponding to the Logic App.
+
+![ApprovalEmail_Deploy_3](Images/ApprovalEmail_Deploy_3.png)
+
+Click on the "**Edit**" button. This will bring us into the Logic Apps Designer.
+
+![ApprovalEmail_Deploy_4](Images/ApprovalEmail_Deploy_4.png)
+
+The step labeled "**Connections**" uses an Office365 connection created during the deployment of this playbook. Before the playbook can be run, this connection will either need to be authorized in the indicated step, or an existing authorized connection may be alternatively selected.
```
Comment (rahul0216) [inline Playbooks/AS-Incident-Response-Approval-Email/README.md:129]: Correct the spelling 'labeled'.

File: Playbooks/AS-Incident-Response-Approval-Email/README.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+The following items are required under the template settings during deployment: 
+
+* An [App Registration](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Incident-Response-Approval-Email#create-an-app-registration) for using the Microsoft Graph API
+* An [Azure Key Vault Secret](https://github.com/Azure/Azure-Sentinel/tree/master/Playbooks/AS-Incident-Response-Approval-Email#create-an-azure-key-vault-secret) containing your App Registration Secret 
+
+
+# 
+### Setup
+                                                                                                                                     
+#### Create an App Registration:
```
Comment (rahul0216) [inline Playbooks/AS-Incident-Response-Approval-Email/README.md:27]: Nicley explained steps for app registration and Azure Key vault setup. 
Can you numbered these steps for clarity wherever applicable?

**Review by rahul0216 — COMMENTED** [review]
Please address the inline comments, and as suggested by Lior update the playbook name.
Rest looks good.

**Comment by v-mchatla** [comment]
Hi @AcceleryntSecurityDev,
Could you please address the Rahul's comments.
Thanks

**Comment by rahul0216** [comment]
@AcceleryntSecurityDev Please share screenshot of successful run of the playbook as well.

File: Playbooks/AS-Incident-Response-Approval-Email/README.md (hunk 2 above)
Comment (AcceleryntSecurityDev) [reply Playbooks/AS-Incident-Response-Approval-Email/README.md]: @rahul0216 I have added numbers to signify the ordering of steps

Comment (AcceleryntSecurityDev) [reply Playbooks/AS-Incident-Response-Approval-Email/README.md]: Here's a screenshot of the testing that was done for this as well:
image

**Review by rahul0216 — APPROVED** [review]
Changes look good.

**Review by v-mchatla — APPROVED** [review]

**Review by v-dvedak — APPROVED** [review]

**Pull request merged**